src/mpyl/utilities/docker/__init__.py

```diff
@@ -146,3 +144,17 @@ def login(logger: Logger, docker_config: DockerConfig) -> None:
     docker.login(server=f'https://{docker_config.host_name}', username=docker_config.user_name,
                  password=docker_config.password)
     logger.debug(f"Logged in as '{docker_config.user_name}'")
+
+
+def create_container(logger: Logger, image_name: str) -> Container:
+    logger.info(f"Creating container from image {image_name}")
+    container = docker.create(image_name)
+    logger.info(f"Created container {container.id}")
+
+    return container
+
+
+def remove_container(logger: Logger, container: Container) -> None:
+    logger.info(f"Removing container {container.id}")
+    docker.remove(container.id)
+    logger.info(f"Removed container {container.id}")
```

Comment:
Maybe these can be debug level

Reply:
I was kinda following the structure of the other functions (`login`, `build`) that also use info logs

Reply:
Guess it's fine like this as well.. though now I'd say those would be better on debug level as well

Reply:
I just checked when & where we use info vs debug and I think `info` here fits into our current overall logging strategy. It's extra information to show/track what is being in the pipeline